app/lib/shared/configuration.dart

```diff
@@ -91,12 +91,13 @@ class EnvConfig {
   final int isolateCount;
 
   EnvConfig._(
-    this.gaeService,
+    String gaeService,
     this.gcloudProject,
     this.gcloudKey,
     this.flutterSdkDir,
     this.isolateCount,
-  ) {
+  )
+      : this.gaeService = gaeService ?? 'default' {
```

Comment:
I think @mkustermann asked me not to have a default value here and we keep being explicit about what service we want to start.

Reply:
@kevmoo This is an unrelated change (again), this change should do only one thing, not several different orthogonal things :-/